src/plants/PlantSelectField.js

```diff
@@ -57,39 +55,52 @@ class PlantSelectField extends React.Component {
           if (error) {
             console.error(error);
           }
-          this.setState({ isLoading: false, showResults: true });
+          this.setState({ ...state, isLoading: false, showResults: true });
```

Comment:
Eu não quero ser exigente aqui então não precisa alterar, mas com o `setState` vc não precisa recriar o state completo com `...state`. Acredito que vc tenha feito isso por estar usando redux ai no momento e isso é um requisito dom redux hehehe

```suggestion
          this.setState({ isLoading: false, showResults: true });
```

Reply:
Tava acontecendo um bug (não lembro qual agora) ai uma das minha hipotéses foi "será que ao chamar o setState sem o spread não tá 'apagando' as outras keys do estado?" ai testei isso e obviamente deu errado hahahaha. Valeu pela dica!